data-plane/receiver/src/test/java/dev/knative/eventing/kafka/broker/receiver/impl/ReceiverVerticleTracingTest.java

```diff
@@ -143,7 +143,6 @@ public void tearDown() throws ExecutionException, InterruptedException {
       .get();
   }
 
-  @Test
```

Comment:
If we don't remove this from here, we can remove the method
```
    @Test
    public void traceIsPropagated() throws ExecutionException, InterruptedException, TimeoutException {
        super.traceIsPropagated();
    }
```
in the subclass

Reply:
This class is `abstract` now, so need to run the tests from the subclass. Or this test not showing In the IDE to run.
I think we need to have the method of subclass.

Reply:
Yes `UnorderedConsumerVerticleTest` with `AbstractConsumerVerticleTest` have the same approach and in my VS-Code IDE they are also not showing as tests that I can run.
I think I should switch to IntelliJ for Java development at least. But my laptop does not allow it, it became so laggy with browser, slack and JetBrains IDE. 😅😅

Reply:
Which VS-code plugin are you using?